common/src/Renderer/Camera.cpp

```diff
@@ -370,7 +370,7 @@ namespace TrenchBroom {
 
         Camera::Camera() :
         m_nearPlane(1.0f),
-        m_farPlane(8000.0f),
```

Comment:
This will decrease depth buffer precision considerably for further distances and might lead to visual artifacts like disappearing edges etc. -- did you check for that? Maybe we need to increase depth buffer precision also, if possible.

Reply:
It looks fine. I tested some 0.125 thick planes (e.g. a large poster on a wall) located at X=8000 and they start z-fighting against the wall behind them at around a camera distance of 1500 units away. It's not very different with the old far clip setting.

This is with a 24-bit depth buffer.